corehq/apps/app_manager/urls.py

```diff
@@ -70,7 +70,7 @@
 urlpatterns = [
     url(r'^$', view_app, name='default_app'),
     url(r'^xform/(?P<form_unique_id>[\w-]+)/$', xform_display, name='xform_display'),
-    url(r'^browse/(?P<app_id>[\w-]+)/modules-(?P<module_id>[\w-]+)/forms-(?P<form_id>[\w-]+)/source/$',
+    url(r'^browse/(?P<app_id>[\w-]+)/(?P<form_unique_id>[\w-]+)/source/$',
```

Comment:
I think this will break for anyone who has tab open during deploy, and they'll just get 400/500s if they try to do anything without reloading. It's probably worth having some way to support both endpoints for the next couple of days